### PR TITLE
Clothing Additions By DakoDemon

### PR DESCRIPTION
**Pull request by ShatteredSwords**
# Description

Added in some more departmental socks, and a few other requested outfits. Did a few bug fixes. - DakoDemon

(I helped solve the merge conflicts by cherry-picking each of these commits and manually going through! Hope I didn't fuck anything up!) -Kota aka ShatteredSwords

---

# Changelog

:cl: DakoDemon
- add: Added a few more outfits
- fix: Fixed the skirt issue for harpies
- fix: Fixed Sec armor values
- tweak: The Autodrobe Restock is now 2000 (from 1730)

### CODE REVIEW

**Comment by ShatteredSwords** [comment]
Alright time to see what I did wrong :P

**Review by Memeji — COMMENTED** [review]
Seems good from what I see.
Only thing I'm wondering about is commenting out the old socks.
Also I believe a fix was applied for Sec values already while I was asleep.

**Comment by ShatteredSwords** [comment]
> Seems good from what I see. Only thing I'm wondering about is commenting out the old socks. Also I believe a fix was applied for Sec values already while I was asleep.

Making the coder and bee socks cheaper because Dako put them in their own PR as a cheaper loadout, so keeping the other loadout would make it redundant

**Comment by ShatteredSwords** [comment]
> Also I believe a fix was applied for Sec values already while I was asleep.

These ones still have a slight armor value as far as I can tell, if you want them coded out I could probably remove them just fine.

**Review by Memeji — APPROVED** [review]
Thanks for the info, overall looks good.

File: Resources/Prototypes/DeltaV/Catalog/VendingMachines/Inventories/pride.yml [hunk 1]
```diff
@@ -27,6 +27,12 @@
        ClothingUniformColorRainbow: 2
        ClothingUnderSocksCoder: 1
        ClothingUnderSocksBee: 1
+       ClothingHandsBeeWarmers: 1
+       ClothingHandsCoderWarmers: 1
+       ClothingUniformThongCoder: 1
+       ClothingUniformThongBee: 1
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/DeltaV/Catalog/VendingMachines/Inventories/pride.yml:33]:
```suggestion
       ClothingHandsBeeWarmers: 1 # Floofstation
       ClothingHandsCoderWarmers: 1 # Floofstation
       ClothingUniformThongCoder: 1 # Floofstation
       ClothingUniformThongBee: 1 # Floofstation
```

File: Resources/Prototypes/DeltaV/Catalog/VendingMachines/Inventories/pride.yml [hunk 2]
```diff
@@ -27,6 +27,12 @@
...
   emaggedInventory:
       # 3 for more friends!
       ClothingUnderSocksCoderValid: 3 # Floofstation
+      ClothingHandsCoderValidWarmers: 3
+      ClothingUniformThongCoderValid: 3
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/DeltaV/Catalog/VendingMachines/Inventories/pride.yml:38]:
```suggestion
      ClothingHandsCoderValidWarmers: 3 # Floofstation
      ClothingUniformThongCoderValid: 3 # Floofstation
```

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/robodrobe.yml [hunk 3]
```diff
@@ -11,3 +11,5 @@
     ClothingHeadsetRobotics: 2
     ClothingOuterWinterRobo: 2
     ClothingShoesBootsWinterRobo: 2 #Delta V: Add departmental winter boots
+    ClothingOuterRoboOveralls: 2 #Floofstation - Add overalls
+    ClothingOuterPlainOveralls: 2
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Catalog/VendingMachines/Inventories/robodrobe.yml:15]:
```suggestion
    ClothingOuterPlainOveralls: 2 # Floofstation
```

File: Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml [hunk 4]
```diff
@@ -54,6 +57,10 @@
     sprite: Floof/Clothing/Departmental/Command/commandwarmer.rsi
   - type: Clothing
     sprite: Floof/Clothing/Departmental/Command/commandwarmer.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Slash: 0.95
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml:63]:
```suggestion
```

File: Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml [hunk 5]
```diff
@@ -12,6 +12,9 @@
     sprite: Floof/Clothing/Departmental/Command/captwarmer.rsi
   - type: Clothing
     sprite: Floof/Clothing/Departmental/Command/captwarmer.rsi
+  - type: GloveHeatResistance
+    heatResistance: 1400
+  - type: Insulated
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml:17]:
```suggestion
```

File: Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml [hunk 6]
```diff
@@ -97,9 +104,12 @@
     sprite: Floof/Clothing/Departmental/Engineering/engiwarmer.rsi
   - type: Clothing
     sprite: Floof/Clothing/Departmental/Engineering/engiwarmer.rsi
+  - type: Insulated
+  - type: GloveHeatResistance
+    heatResistance: 1400
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml:109]:
```suggestion
```

File: Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml [hunk 7]
```diff
@@ -127,6 +137,49 @@
...
+  name: chief engineer's arm-warmers
+  description: A pair of rather nice arm-warmers, styled for Engineering roles.
+  components:
+  - type: Sprite
+    sprite: Floof/Clothing/Departmental/Engineering/chiefengiwarmer.rsi
+  - type: Clothing
+    sprite: Floof/Clothing/Departmental/Engineering/chiefengiwarmer.rsi
+  - type: Insulated
+  - type: GloveHeatResistance
+    heatResistance: 1400
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml:153]:
```suggestion
```

File: Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml [hunk 8]
```diff
@@ -156,6 +213,10 @@
     sprite: Floof/Clothing/Departmental/Epistemics/episocks.rsi
   - type: Clothing
     sprite: Floof/Clothing/Departmental/Epistemics/episocks.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Caustic: 0.85
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml:219]:
```suggestion
```

File: Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml [hunk 9]
```diff
@@ -167,6 +228,10 @@
     sprite: Floof/Clothing/Departmental/Epistemics/epithong.rsi
   - type: Clothing
     sprite: Floof/Clothing/Departmental/Epistemics/epithong.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Caustic: 0.95
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml:234]:
```suggestion
```

File: Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml [hunk 10]
```diff
@@ -235,6 +300,121 @@
...
+  description: A pair of rather nice arm-warmers, styled for the research director.
+  components:
+  - type: Sprite
+    sprite: Floof/Clothing/Departmental/Epistemics/directorepiwarmer.rsi
+  - type: Clothing
+    sprite: Floof/Clothing/Departmental/Epistemics/directorepiwarmer.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Caustic: 0.85
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml:317]:
```suggestion
```

File: Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml [hunk 11]
```diff
@@ -235,6 +300,121 @@
...
+  description: A pair of thigh-high socks, styled for the research director.
+  components:
+  - type: Sprite
+    sprite: Floof/Clothing/Departmental/Epistemics/directorepisocks.rsi
+  - type: Clothing
+    sprite: Floof/Clothing/Departmental/Epistemics/directorepisocks.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Caustic: 0.85
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml:336]:
```suggestion
```

File: Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml [hunk 12]
```diff
@@ -235,6 +300,121 @@
...
+  description: Oddly, styled for the research director, but is this even considered a piece of clothing?
+  components:
+  - type: Sprite
+    sprite: Floof/Clothing/Departmental/Epistemics/directorepithong.rsi
+  - type: Clothing
+    sprite: Floof/Clothing/Departmental/Epistemics/directorepithong.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Caustic: 0.85
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml:351]:
```suggestion
```

File: Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml [hunk 13]
```diff
@@ -235,6 +300,121 @@
...
+    sprite: Floof/Clothing/Departmental/Epistemics/mantisepiwarmer.rsi
+  - type: Clothing
+    sprite: Floof/Clothing/Departmental/Epistemics/mantisepiwarmer.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.95
+        Slash: 0.95
+        Piercing: 0.95
+        Heat: 0.95
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml:373]:
```suggestion
```

File: Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml [hunk 14]
```diff
@@ -235,6 +300,121 @@
...
+    sprite: Floof/Clothing/Departmental/Epistemics/mantisepisocks.rsi
+  - type: Clothing
+    sprite: Floof/Clothing/Departmental/Epistemics/mantisepisocks.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.95
+        Slash: 0.95
+        Piercing: 0.95
+        Heat: 0.95
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml:395]:
```suggestion
```

File: Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml [hunk 15]
```diff
@@ -235,6 +300,121 @@
...
+    sprite: Floof/Clothing/Departmental/Epistemics/mantisepithong.rsi
+  - type: Clothing
+    sprite: Floof/Clothing/Departmental/Epistemics/mantisepithong.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.95
+        Slash: 0.95
+        Piercing: 0.95
+        Heat: 0.95
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml:413]:
```suggestion
```

File: Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml [hunk 16]
```diff
@@ -278,6 +458,92 @@
...
+  description: A pair of thigh-high socks, styled for Couriers.
+  components:
+  - type: Sprite
+    sprite: Floof/Clothing/Departmental/Logistics/mailcarriersocks.rsi
+  - type: Clothing
+    sprite: Floof/Clothing/Departmental/Logistics/mailcarriersocks.rsi
+  - type: ClothingSpeedModifier
+    walkModifier: 1
+    sprintModifier: 1.2
+    enabled: true
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml:531]:
```suggestion
```

File: Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml [hunk 17]
```diff
@@ -321,6 +587,145 @@
...
+  description: A pair of rather nice arm-warmers, styled for Chemists.
+  components:
+  - type: Sprite
+    sprite: Floof/Clothing/Departmental/Medical/chemistwarmer.rsi
+  - type: Clothing
+    sprite: Floof/Clothing/Departmental/Medical/chemistwarmer.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Caustic: 0.95
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml:604]:
```suggestion
```

File: Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml [hunk 18]
```diff
@@ -321,6 +587,145 @@
...
+  description: A pair of thigh-high socks, styled for Chemists.
+  components:
+  - type: Sprite
+    sprite: Floof/Clothing/Departmental/Medical/chemistsocks.rsi
+  - type: Clothing
+    sprite: Floof/Clothing/Departmental/Medical/chemistsocks.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Caustic: 0.95
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml:623]:
```suggestion
```

File: Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml [hunk 19]
```diff
@@ -321,6 +587,145 @@
...
+  description: Styled for a Chemists, but is this even considered a piece of clothing?
+  components:
+  - type: Sprite
+    sprite: Floof/Clothing/Departmental/Medical/chemistthong.rsi
+  - type: Clothing
+    sprite: Floof/Clothing/Departmental/Medical/chemistthong.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Caustic: 0.95
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml:638]:
```suggestion
```

File: Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml [hunk 20]
```diff
@@ -321,6 +587,145 @@
...
+  description: A pair of thigh-high socks, styled for Paramedic roles.
+  components:
+  - type: Sprite
+    sprite: Floof/Clothing/Departmental/Medical/paramedsocks.rsi
+  - type: Clothing
+    sprite: Floof/Clothing/Departmental/Medical/paramedsocks.rsi
+  - type: ClothingSpeedModifier
+    walkModifier: 1
+    sprintModifier: 1.2
+    enabled: true
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml:672]:
```suggestion
```

File: Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml [hunk 21]
```diff
@@ -336,12 +741,18 @@
...
+    fiberColor: fibers-black
   - type: FingerprintMask
-  - type: GroupExamine
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.95
+        Slash: 0.95
+        Piercing: 0.95
+        Heat: 0.95
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml:752]:
```suggestion
```

File: Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml [hunk 22]
```diff
@@ -350,7 +761,13 @@
...
   - type: Clothing
     sprite: Floof/Clothing/Departmental/Security/secsocks.rsi
-  - type: GroupExamine
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.95
+        Slash: 0.95
+        Piercing: 0.95
+        Heat: 0.95
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml:770]:
```suggestion
```

File: Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml [hunk 23]
```diff
@@ -362,7 +779,137 @@
...
   - type: Clothing
     sprite: Floof/Clothing/Departmental/Security/secthong.rsi
-  - type: GroupExamine
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.95
+        Slash: 0.95
+        Piercing: 0.95
+        Heat: 0.95
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml:788]:
```suggestion
```

File: Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml [hunk 24]
```diff
@@ -362,7 +779,137 @@
...
+    fiberMaterial: fibers-durathread
+    fiberColor: fibers-black
+  - type: FingerprintMask
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.9
+        Slash: 0.9
+        Piercing: 0.9
+        Heat: 0.9
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml:814]:
```suggestion
```

File: Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml [hunk 25]
```diff
@@ -362,7 +779,137 @@
...
+    sprite: Floof/Clothing/Departmental/Security/hossocks.rsi
+  - type: Clothing
+    sprite: Floof/Clothing/Departmental/Security/hossocks.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.9
+        Slash: 0.9
+        Piercing: 0.9
+        Heat: 0.9
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml:832]:
```suggestion
```

File: Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml [hunk 26]
```diff
@@ -362,7 +779,137 @@
...
+    sprite: Floof/Clothing/Departmental/Security/hosthong.rsi
+  - type: Clothing
+    sprite: Floof/Clothing/Departmental/Security/hosthong.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.9
+        Slash: 0.9
+        Piercing: 0.9
+        Heat: 0.9
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml:850]:
```suggestion
```

File: Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml [hunk 27]
```diff
@@ -362,7 +779,137 @@
...
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.95
+        Slash: 0.95
+        Piercing: 0.95
+        Heat: 0.9
+  - type: GuideHelp
+    guides:
+    - Forensics
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml:876]:
```suggestion
```

File: Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml [hunk 28]
```diff
@@ -362,7 +779,137 @@
...
+  - type: Tag #DeltaV, needed for species with nonhuman legs/can only wear skirts
+    tags:
+    - Skirt
+
+  #Detective
+- type: entity
+  parent: ClothingHandsBase
+  id: ClothingHandsDetectiveWarmers
+  name: detective's arm-warmers
+  description: A pair of rather nice arm-warmers, styled for a Detective. Does not leave Fibers or Fingerprints.
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml:860]:
```suggestion
  description: A pair of rather nice arm-warmers, styled for a Detective.
```

File: Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml [hunk 29]
```diff
@@ -362,7 +779,137 @@
...
+  components:
+  - type: Sprite
+    sprite: Floof/Clothing/Departmental/Security/detectwarmer.rsi
+  - type: Clothing
+    sprite: Floof/Clothing/Departmental/Security/detectwarmer.rsi
+  - type: FingerprintMask
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml:866]: This should not replace the detective equipement and so should have fiber.

File: Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml [hunk 30]
```diff
@@ -362,7 +779,137 @@
...
+    sprite: Floof/Clothing/Departmental/Security/detectsocks.rsi
+  - type: Clothing
+    sprite: Floof/Clothing/Departmental/Security/detectsocks.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.95
+        Slash: 0.95
+        Piercing: 0.95
+        Heat: 0.9
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml:894]:
```suggestion
```

File: Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml [hunk 31]
```diff
@@ -362,7 +779,137 @@
...
+    sprite: Floof/Clothing/Departmental/Security/detectthong.rsi
+  - type: Clothing
+    sprite: Floof/Clothing/Departmental/Security/detectthong.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.95
+        Slash: 0.95
+        Piercing: 0.95
+        Heat: 0.9
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Clothing/departmental_clothes.yml:912]:
```suggestion
```

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]
Those clothing must NOT replace security, protective, job equipement and so should in any cases not have speed boosts or armor or special effects, keep them simple.

If someone need to have more project if handling something dangerous they should get protective gear for that moment.

**Comment by ShatteredSwords** [comment]
I'm gonna resolve these on PC so I don't have to handle a dying phone while merging the suggestions lol

**Comment by ShatteredSwords** [comment]
So turns out github will test every single commit before testing the whole thing together. I ate all of the actions tests. Do not do what I did and learn the hard way, any contributor that may read this.

**Review by FoxxoTrystan — APPROVED** [review]

**Review by Memeji — APPROVED** [review]

**Pull request merged**